plugins/source/aws/resources/services/qldb/ledger_journal_s3_exports.go

```diff
@@ -18,7 +18,7 @@ func ledgerJournalS3Exports() *schema.Table {
 		Name:        tableName,
 		Description: `https://docs.aws.amazon.com/qldb/latest/developerguide/API_JournalS3ExportDescription.html`,
 		Resolver:    fetchQldbLedgerJournalS3Exports,
-		Transform:   transformers.TransformWithStruct(&types.JournalS3ExportDescription{}),
+		Transform:   transformers.TransformWithStruct(&types.JournalS3ExportDescription{}, transformers.WithPrimaryKeys("ExportId")),
```

Comment:
Should we also include the ledger ARN or does AWS document that the ExportId is globally unique?

Reply:
All we know is it's a UUID. Adding `arn` just in case.